mappings/servers/akyto/metadata.json

```diff
@@ -0,0 +1,34 @@
+{
+    "id": "akyto",
+    "name": "akyto",
```

Comment:
This should probably be Akyto Network to be case sensitive in accordance to your logo

Reply:
can't you just say it all at once instead of busting balls, man?